art-community-platform/backend/app/user/views.py

```diff
@@ -16,6 +16,8 @@ def health_check(request):
 def signup(req):
     data = json.loads(req.body)
     try:
+        name = data['name']
+        birthdate = data['birthdate']
```

Comment:
Parameters assigned in this try-except block are mandatory fields, therefore except block is returning Http Bad Request with message "Missing Fields". This means name and birth date fields will be mandatory as planned.